documentation/app/templates/application.hbs

```diff
@@ -2,8 +2,70 @@
   Copyright (c) HashiCorp, Inc.
 }}
 
-{{page-title "TestApp"}}
+{{page-title "Consul UI Toolkit"}}
 
-<h2 id="title">Welcome to Ember</h2>
+<div class="application">
+  <Hds::SideNav::Wrapper>
```

Comment:
Can we make the side nav fixed, so as the body content scrolls the nav stays in place?

Reply:
Ha whoops!